src/routes/github/branch/github-branch-get.ts

```diff
@@ -1,29 +1,43 @@
 import { Request, Response } from "express";
-import { createUserClient } from "utils/get-github-client-config";
+import { createInstallationClient } from "utils/get-github-client-config";
+import { Subscription } from "models/subscription";
+import { getLogger } from "config/logger";
 
-// Get a branch for the **USER TOKEN**
 export const GithubBranchGet = async (req: Request, res: Response): Promise<void> => {
 	const {
 		jiraHost,
-		githubToken,
 		gitHubAppConfig
 	} = res.locals;
 	const { owner, repo, ref } = req.params;
 
-	if (!githubToken || !gitHubAppConfig) {
+	const logger = getLogger("github-branch-get", {
+		fields: req.log?.fields
+	});
+
+	if (!gitHubAppConfig) {
+		logger.warn("No gitHubAppConfig found.");
```

Comment:
We should probably update our `Errors` enum to include MISSING_GITHUB_APP_CONFIG and MISSING_SUBSCRIPTION so we can start ensuring our warn/error messaging is consistent across the app.